remoulade/middleware/time_limit.py

```diff
@@ -77,25 +91,29 @@ def _handle(self, *_) -> None:
                             "cancel message and exit the worker",
                             message.message_id,
                         )
-                        raise SystemExit(1)
                     except NoCancelBackend:
                         # Sending SIGKILL would requeue the message via RabbitMQ
```

Comment:
maybe remove this comment, there should be somthing about https://www.rabbitmq.com/docs/quorum-queues#poison-message-handling thisto be activated if you don't want a long message to be redeliverred infinitely.

Reply:
most likely we will be able to cancel the mesage, this is an edge case of an edge case in which case it will make a worker suicide in loop so we will end up doing something about it i think